edgelet/aziot-edged/src/product_info.rs

```diff
@@ -0,0 +1,194 @@
+use std::convert::AsRef;
+use std::fmt;
+use std::fs;
+use std::io::{self, BufRead};
+use std::path::Path;
+use std::process::{Command, Output};
+
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize, PartialEq)]
+pub(crate) struct ProductInfo {
```

Comment:
This should be merged with the `OsInfo` in `aziotctl check` in some way. This is basically a super-set of `OsInfo`, and this extra info would make sense to have in the `check` output.json too.